src/rules/converters/eslint-plugin-react/jsx-key.ts

```diff
@@ -0,0 +1,13 @@
+import { RuleConverter } from "../../converter";
+
+export const convertJsxKey: RuleConverter = () => {
+    return {
+        rules: [
+            {
+                ruleName: "react/jsx-key",
+                ruleArguments: ["<enabled>"],
```

Comment:
Yeah, looking through the [source](https://github.com/palantir/tslint-react/blob/b92fb075375394f84594853b928e670d729a1ed8/src/rules/jsxKeyRule.ts#L59) for the TSLint side, they don't account for JSX fragments either. So I guess the rule should have no rule arguments either, until the next major version of `eslsint-plugin-react`.